From 0ac43410bc3c7f48df576dd50e9a2d9294b84aed Mon Sep 17 00:00:00 2001
From: Edward Hervey <edward@centricular.com>
Date: Wed, 21 Feb 2018 09:05:45 +0100
Subject: [PATCH] agent: Redefine socket error messages for windows

In the same way we do it for the other error messages
---
 agent/pseudotcp.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/agent/pseudotcp.h b/agent/pseudotcp.h
index e7c8eaa..687c459 100644
--- a/agent/pseudotcp.h
+++ b/agent/pseudotcp.h
@@ -78,6 +78,14 @@
 #ifndef ECONNRESET
 #  define ECONNRESET WSAECONNRESET
 #endif
+
+#ifndef EMSGSIZE
+#  define EMSGSIZE WSAEMSGSIZE
+#endif
+
+#ifndef ETIMEDOUT
+#  define ETIMEDOUT WSAETIMEDOUT
+#endif
 #endif
 #endif
 
-- 
2.14.3

